keras/layers/convolutional.py

```diff
@@ -1449,7 +1464,17 @@ class ZeroPadding2D(Layer):
     # Arguments
         padding: tuple of int (length 2)
             How many zeros to add at the beginning and end of
-            the 2 padding dimensions (axis 3 and 4).
+            the 2 padding dimensions (rows and cols) symmetrically.
+            Setting padding to the values different from (1, 1)
+            overrides top_pad, bottom_pad; and left_pad, right_pad arguments.
```

Comment:
@fchollet One question, if I'll leave an option to pass the padding as a tuple of length 2, it should also be left as a default argument of value (1, 1). But in this case it will contradict with the defaults for the dictionary (missing -> zero). I would set defaults for the dict to 1's too, I can change later if you wouldn't agree.